composer.json

```diff
@@ -36,10 +37,14 @@
     },
     "autoload": {
         "psr-0": {
+            "TYPO3\\Flow\\Log\\": "Neos.Flow.Log/Classes",
             "TYPO3\\Eel": "TYPO3.Eel/Classes",
             "TYPO3\\Flow": "TYPO3.Flow/Classes",
             "TYPO3\\Fluid": "TYPO3.Fluid/Classes",
             "TYPO3\\Kickstart": "TYPO3.Kickstart/Classes"
+        },
+        "psr-4": {
+            "Neos\\Flow\\Log\\": "Neos.Flow.Log/Classes"
```

Comment:
Yes, nothing speaks against that and I wanted to prevent WTF moments when we actually want to transfer the classes to a new namespace. Basically we could already create dummy classes extending the old ones (or vice versa) and it would work.